### DISCUSSION TITLE
tested ESP32-C3

### DISCUSSION THREAD

**Discussion by det0ga** [tttapa/Control-Surface#1032] [General]
I tested the midi over BLE and rtpMidi functionality of the library on my Xiao ESP32-C3.
I tested Input and Output of midi signals as well as pipes and everything worked as expected.

Could you also add support for nRF52? More specifically the Xiao nRF52840

**Comment by tttapa** [comment]
Thanks for trying it out and reporting back!

I've just pushed 99d1e0f651461ad641829eb6c6b4ba0501c07e22, which allows you to add `#define CS_USE_ARDUINOBLE 1` before including `<Control_Surface.h>`. This will tell Control Surface to enable MIDI over BLE support using the ArduinoBLE library.
